src/main/java/com/google/devtools/build/lib/remote/options/RemoteOptions.java

```diff
@@ -433,6 +435,33 @@ public RemoteOutputsStrategyConverter() {
               + " discard the remotely cached values if they don't match the expected value.")
   public boolean remoteVerifyDownloads;
 
+  @Option(
+      name = "remote_keepalive_time",
+      defaultValue = "null",
+      documentationCategory = OptionDocumentationCategory.REMOTE,
+      effectTags = {OptionEffectTag.UNKNOWN},
+      help =
+          "Sets the time without read activity before sending a keepalive ping. Clients must "
+              + "receive permission from the service owner before enabling this option. Keepalives "
+              + "can increase the load on services and are commonly \"invisible\" making it hard "
+              + "to notice when they are causing excessive load. Clients are strongly encouraged "
+              + "to use only as small of a value as necessary.")
+  @Nullable
+  public Duration remoteKeepaliveTime;
+
+  @Option(
+      name = "remote_keepalive_timeout",
+      defaultValue = "null",
+      documentationCategory = OptionDocumentationCategory.REMOTE,
+      effectTags = {OptionEffectTag.UNKNOWN},
+      help =
+          "Sets the time waiting for read activity after sending a keepalive ping. If the time "
+              + "expires without any read activity on the connection, the connection is considered "
+              + "dead. This value should be at least multiple times the RTT to allow for lost "
```

Comment:
Do not use abbreviations - they may not be universally clear to all readers.

Reply:
Agreed!